SourceSinkPushPull/locale/ru/base.cfg

```diff
@@ -38,18 +38,18 @@ sspp-request-io=Ввод/вывод запроса
 
 [item-description]
 sspp-stop=Усовершенствованная остановка поезда.
-sspp-general-io=Разместите возле остановки поезда SSPP.
-sspp-provide-io=Разместите возле остановки поезда SSPP.
-sspp-request-io=Разместите возле остановки поезда SSPP.
+sspp-general-io=Разместите возле остановки поезда "SSPP".
+sspp-provide-io=Разместите возле остановки поезда "SSPP".
+sspp-request-io=Разместите возле остановки поезда "SSPP".
 
 [entity-description]
 sspp-stop=Выход: Содержимое поезда
 sspp-general-io=Вход: Груз в хранилище
-sspp-provide-io=Выход: Груз, подлежащий загрузке
-sspp-request-io=Выход: Груз, подлежащий выгрузке
+sspp-provide-io=Выход: Груз для погрузки

```

Comment:
missing comma

Reply:
or maybe extra comma on request io? I feel like either both should have commas or neither

Reply:
nope ignore this, it looks like what you did makes sense haha, it just looked weird to me seeing a comma in only one